packages/slice-machine/src/pages/changes.tsx

```diff
@@ -155,6 +171,28 @@ const Changes: React.FunctionComponent = () => {
                 void onPush(false); // not deleting documents by default
               }}
             />
+            <Toast
+              anchor={<Box position="fixed" top={30} right={32} />}
+              open={isToastOpen}
+              onOpenChange={setIsToastOpen}
+              variant="card"
+              seconds={20}
+              title={
+                <Box flexDirection="column" alignItems="flex-start" as="span">
```

Comment:
why `span`?

Reply:
this goes inside `Text` component in the Toast which is rendered as `p` and this caused some warnings showing up in console

Reply:
in that case, i would talk with design to align on this

we're workarounding the _title_ (that originally seemed to mean for a paragraph, so we could argue that `title` here is misleading) to show a header and a body

maybe those can be two different props

Reply:
Originally Toast accepted a ReactNode as a title, i didn't want to add too many new props too it, so I used that. 
But you're right, this might be confusing. There is [open PR](https://github.com/prismicio/editor/pull/1193) for few more tweaks to the Toast, so I can still change that

Reply:
yeah, so i would first align with design on this, and then post a message in #frontend-engineering about the best way to tackle this:
- have an unopinionated react node so the consumer can set whatever
- have a heading and a body (both optional, some optional...?)
- something else?